dbt-cta/empower/models/1_cta_full_refresh/ctaResults_answerIdsByPromptId_base.sql

```diff
@@ -0,0 +1,31 @@
+{% set partitions_to_replace = [
+    'timestamp_trunc(current_timestamp, day)',
+    'timestamp_trunc(timestamp_sub(current_timestamp, interval 1 day), day)'
+] %}
+{{ config(
```

Comment:
All of the `1_cta_full_refresh` models should have `partitions = partitions_to_replace` in the config

Reply:
This is my only flag! Change this and I believe it shoule be ok.

Reply:
ah!! thanks for flagging y'all - I wen ahead and redid these using Jesus' script and I think these are all fixed now!!